package.json

```diff
@@ -38,25 +38,18 @@
   "scripts": {
     "benchmark-rule": "node scripts/benchmark-rule.js",
     "format": "prettier . --write",
-    "postinstall": "is-ci || husky install",
+    "prepare": "husky install",
```

Comment:
[note] The husky document uses `prepare` instead of `postinstall`:
<https://typicode.github.io/husky/#/?id=install>

Reply:
Do we still need to do something to [disable hooks on the CI](https://typicode.github.io/husky/#/?id=disable-hooks-in-ci)?

Perhaps:

```
# .husky/pre-commit
# ...
[ -n "$CI" ] && exit 0
```

Reply:
Good catch! Fixed via 57eeb86d.

I think we now don't need the `CI` check for the `prepare` script via `is-ci`:

```json
{
  "scripts": {
    "prepare": "is-ci || husky install"
  }
}
```

If we would need it in the future, then let's install it again.